module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <array>
+#include <cstdint>
+#include <utility>
```

Comment:
Why do you need utility include in the header? Generally you want to have as little includes as possible in header in favour of cpp includes

Reply:
not a scope of the offer, corrected on past branch